pyinteraph/path_analysis.py

```diff
@@ -5,6 +5,11 @@
 import MDAnalysis as mda
 import itertools
 import re
+import matplotlib.pyplot as plt
+import seaborn as sns
+from matplotlib.colors import ListedColormap
+from matplotlib.colors import LinearSegmentedColormap
```

Comment:
you can import both classes from `matplotlib.colors` in a single import statement